js/source/raster_tile_source.js

```diff
@@ -37,15 +40,27 @@ class RasterTileSource extends Evented {
         this.map = map;
     }
 
+    setBounds(bounds) {
+      this.bounds = bounds;
+      if (bounds) {
+        this.tileBounds = new TileBounds(bounds, this.minzoom, this.maxzoom);
+      }
+    }
+
     serialize() {
         return {
             type: 'raster',
             url: this.url,
             tileSize: this.tileSize,
-            tiles: this.tiles
+            tiles: this.tiles,
+            bounds: this.bounds,
```

Comment:
You're right, I was tripped up by the naming of `bounds` vs `tileBounds`. Curious if there's a quick fix that would make this clearer, perhaps `serializedBounds`?